roles/include_components/tasks/track_repo_commit_component.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Assert ic_commit_url
+  assert:
+    that:
+      # Assert that ic_commit_url is a valid URL
+      # of the sort https://hosting.com/organisation/reponame/commit/commit_hash
+      - ic_commit_url is match('^https://[^/]+/[^/]+/[^/]+/commit/[0-9a-f]{40}$')
```

Comment:
Not sure if all the commit URLs follow the same pattern :worried: 

https://gitlab.com/wireshark/wireshark/-/commit/b29a96913167e7ecf9512d32a87baa7539ed0944
https://softwarefactory-project.io/r/plugins/gitiles/dci-openshift-agent/+/6fe7093548ccbcf45884d62f34cfd79317ff09e3
https://src.fedoraproject.org/rpms/kernel/c/78dae7034132013adc5528d6d5f2ba8415c5edb7


Maybe this change is only useful for GH? :octocat:

Reply:
nice catch, let's go with the gh components then, for the moment